Dockerfile

```diff
@@ -9,16 +9,21 @@ LABEL go.version=${GO_VER}
 LABEL ginkgo.version=${GINKGO_VER}
 LABEL container.user=${CONTAINERUSER}
 
-ENV PATH "$PATH:/usr/local/go/bin:/root/go/bin"
+ENV GOPATH="/usr/local/go"
+ENV PATH "$PATH:$GOPATH:$GOPATH/bin"
```

Comment:
do we anticipate having executables under $GOPATH 
also probably better to have $GOPATH/bin before $PATH, i.e.:
ENV PATH "$GOPATH/bin:$PATH"